src/components/blog/BlogHeader.jsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+
+export default function BlogHeader(props) {
+  const latestPost = props.posts[0];
```

Comment:
BlogHeader is unclear whether this is the header of a single blog post or the header of all blogs, im assuming allblogs since you're doing posts[0] but it either needs to be renamed (the component) or it needs to have better comments